common/content-api.js

```diff
@@ -234,35 +234,30 @@ function getRecentFundingProgrammes({ locale, limit = 3 }) {
     });
 }
 
-function getFundingProgramme({
-    locale,
-    slug,
-    previewMode = false,
-    query = {}
-}) {
+function getFundingProgramme({ locale, slug, query = {}, requestParams = {} }) {
     return fetch(`/v2/${locale}/funding-programmes/${slug}`, {
-        qs: addPreviewParams(previewMode, { ...query })
+        qs: addPreviewParams(requestParams, { ...query })
     }).then(response => get('data.attributes')(response));
 }
 
 function getResearch({
     locale,
     slug = null,
-    previewMode = null,
     query = {},
+    requestParams = {},
     type = null
 }) {
     if (slug) {
         return fetch(`/v1/${locale}/research/${slug}`, {
-            qs: addPreviewParams(previewMode, { ...query })
+            qs: addPreviewParams(requestParams, { ...query })
```

Comment:
Merging `requestParams` with `query` is a bit redundant as `query` will be a superset of `requestParams`. Not a big deal for now but we should come back and clean this up.

Reply:
Hmm true, in this instance the only thing being passed through is the `req.query.social` param, which is now "global" via these changes – meaning I can remove it everywhere using these calls.